openage-masterserver.cabal

```diff
@@ -13,28 +13,78 @@ license:             AGPL-3
 license-file:        legal/AGPLv3
 author:              Jonas Jelten
 maintainer:          jj@sft.mx
-copyright:           2015, openage team
+copyright:           2016, openage team
 category:            Network, Game
 build-type:          Simple
 extra-source-files:  README.md
 cabal-version:       >=1.10
 
-executable openage-masterserver
-  ghc-options:         -O2 -Wall -dynamic -threaded -fno-warn-missing-signatures -fwarn-tabs
+library
+  ghc-options:         -O2 -Wall -Werror -dynamic -threaded -fwarn-tabs -fwarn-incomplete-uni-patterns -fwarn-incomplete-record-updates
```

Comment:
For portability reasons, `-Werror` is a bad idea, compiler updates might render the project unusable that way.